tests/pytests/test.py

```diff
@@ -4371,6 +4371,8 @@ def test_with_tls_and_non_tls_ports():
     # Upon setting `tls-cluster` to `no`, we should still be able to succeed
     # connecting the coordinator to the shards, just not in TLS mode.
     run_command_on_all_shards(env, 'CONFIG', 'SET', 'tls-cluster', 'no')
+    time.sleep(2)   # Wait a bit for the new topology to kick in (inter-node
```

Comment:
No redis command for waiting for these changes? (Like `wait` for replicas)

Reply:
Maybe we can use `search.CLUSTERINFO` to see if we got the new topology.
I'll see if there's a good field for that (probably not).

Reply:
Yup, done.